src/main/java/models/EnglishGame.java

```diff
@@ -0,0 +1,28 @@
+package models;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Christian on 2/16/2016.
+ */
+public class EnglishGame extends Game {
+
+    // Constructor for EnglishGame
+    public EnglishGame() {
+        cols.add(new ArrayList<Card>());
+        cols.add(new ArrayList<Card>());
+        cols.add(new ArrayList<Card>());
+        cols.add(new ArrayList<Card>());
+        error = false;
+    }
```

Comment:
The constructor actually works the same between both of the subclasses, this can be left in the super class.